src/object/cli_obj.c

```diff
@@ -496,92 +496,131 @@ obj_shard_tgts_query(struct dc_object *obj, uint32_t map_ver, uint32_t shard,
 	shard_tgt->st_rank	= obj_shard->do_target_rank;
 	shard_tgt->st_shard	= shard,
 	shard_tgt->st_tgt_idx	= obj_shard->do_target_idx;
+	shard_tgt->st_tgt_id	= obj_shard2tgtid(obj, shard);
 	obj_shard_close(obj_shard);
 
 out:
 	return rc;
 }
 
+#define OBJ_TGT_INLINE_NR	(1)
```

Comment:
why it's defined to 1?

Reply:
with more than 1 it will exceed task embedded space - tse_task_stack_push will fail for IO, as now pushed more to task default embedded space.
If want to use more space, it needs to enlarge the default task space size.
One inline is useful for quite some cases for example srv-dispatch IO or fetch only need one leader, we can enlarge task space and OBJ_TGT_INLINE_NR  later when needed.